sotodlib/io/g3tsmurf_db.py

```diff
@@ -125,7 +125,7 @@ class Observations(Base):
         across stream_ids.
     timing : bool
         If true, the files of the entry observation were made with times
-        referenced to the timing system.
+        referenced to the external timing system and high precision timestamps.
```

Comment:
nitpicky, but "with times downsampled using the external timing system, and high precision timestamps"

With downsample_mode='internal', timing is still referenced to external timing system, but downsampled data is not aligned